.yarnrc.yml

```diff
@@ -1,9 +1,12 @@
 nodeLinker: node-modules
 
-## Ikke legg inn token her
```

Comment:
Dette ble fjernet automatisk, og det har en sammenheng med at de egentlig vil at man skal bruke `yarn set ` i stedet for å endre filen manuelt